internal/analyzer/analyzer.go

```diff
@@ -64,25 +62,21 @@ func (a *Analyzer) Analyze(commits []shared.Commit) map[shared.Release][]shared.
 
 	for _, commit := range commits {
 		for _, rule := range a.analyzeCommits.getRules() {
```

Comment:
Maybe we can test all rules at once via regex, would improve the speed a little bit. But this we can do in an extra PR